satellite/console/consoleweb/server.go

```diff
@@ -62,6 +63,8 @@ func NewServer(logger *zap.Logger, config Config, service *console.Service, list
 		listener: listener,
 	}
 
+	log.Printf("Starting Satellite UI...")
```

Comment:
You can use the `logger` that was passed to the constructor, i.e. `logger.Debug(...)`.

Reply:
Updated.

Reply:
There is no `s` at this point. It should be `logger.Debug(...)` instead of `s.log.Debug(...)`.